[2022-06-20 06:36:59,416] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dags_new_version.create_table scheduled__2022-06-17T00:00:00+00:00 [queued]>
[2022-06-20 06:36:59,457] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dags_new_version.create_table scheduled__2022-06-17T00:00:00+00:00 [queued]>
[2022-06-20 06:36:59,466] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 06:36:59,469] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2022-06-20 06:36:59,471] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 06:36:59,589] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-06-17 00:00:00+00:00
[2022-06-20 06:36:59,605] {standard_task_runner.py:52} INFO - Started process 183 to run task
[2022-06-20 06:36:59,624] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dags_new_version', 'create_table', 'scheduled__2022-06-17T00:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/user.py', '--cfg-path', '/tmp/tmpgrpbxvcs', '--error-file', '/tmp/tmpzu15yqve']
[2022-06-20 06:36:59,632] {standard_task_runner.py:80} INFO - Job 165: Subtask create_table
[2022-06-20 06:36:59,817] {task_command.py:370} INFO - Running <TaskInstance: dags_new_version.create_table scheduled__2022-06-17T00:00:00+00:00 [running]> on host ca3ce831c04f
[2022-06-20 06:37:00,087] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dags_new_version
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-17T00:00:00+00:00
[2022-06-20 06:37:00,115] {base.py:68} INFO - Using connection ID 'postgre_sql' for task execution.
[2022-06-20 06:37:00,135] {dbapi.py:213} INFO - Running statement: 
        DROP TABLE IF EXISTS z1_user;
        CREATE TABLE z1_user(
            user_id SERIAL PRIMARY KEY,
            firstname VARCHAR(255),
            lastname VARCHAR(255),
            country VARCHAR(255),
            username VARCHAR(255),
            password VARCHAR(255),
            email VARCHAR(255)
        )
        , parameters: None
[2022-06-20 06:37:00,231] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dags_new_version, task_id=create_table, execution_date=20220617T000000, start_date=20220620T063659, end_date=20220620T063700
[2022-06-20 06:37:00,328] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-20 06:37:00,406] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
